### ISSUE TITLE
Crashed when removing the bird from the cage

### ISSUE THREAD

**Issue by yue-git** [penguin0616/insight#16]
Here is the crash log.
```
[00:10:44]: ...rkshop-2081254154/scripts/descriptors/perishable.lua:212: attempt to call method 'GetOwner' (a nil value)
LUA ERROR stack traceback:
E:/games/stream/steamapps/common/dont_starve/data/../mods/workshop-2081254154/scripts/descriptors/perishable.lua:212 in (upvalue) GetPerishData (Lua) <196-309>
   self =
      perishtime = 960
      perishremainingtime = 801.89649625816
      LongUpdate = function - @E:/games/stream/steamapps/common/dont_starve/data/../mods/workshop-584259222/perishable/rog.lua:80
      updatetask = PERIODIC 113486: 10.204661
      StartPerishing = function - @E:/games/stream/steamapps/common/dont_starve/data/../mods/workshop-584259222/perishable/rog.lua:87
      newUpdate = function - @E:/games/stream/steamapps/common/dont_starve/data/../mods/workshop-584259222/perishable/rog.lua:2
      inst = 113486 - robin_winter(LIMBO) (valid:true)
      frozenfiremult = false
      perishfn = function - @E:/games/stream/steamapps/common/dont_starve/data/scripts/standardcomponents.lua:438
   next_task_call = 9.3000004850328
   inst = 113486 - robin_winter(LIMBO) (valid:true)
   is_critter = false
E:/games/stream/steamapps/common/dont_starve/data/../mods/workshop-2081254154/scripts/descriptors/perishable.lua:328 in (field) Describe (Lua) <311-345>
   self =
      perishtime = 960
      perishremainingtime = 801.89649625816
      LongUpdate = function - @E:/games/stream/steamapps/common/dont_starve/data/../mods/workshop-584259222/perishable/rog.lua:80
      updatetask = PERIODIC 113486: 10.204661
      StartPerishing = function - @E:/games/stream/steamapps/common/dont_starve/data/../mods/workshop-584259222/perishable/rog.lua:87
      newUpdate = function - @E:/games/stream/steamapps/common/dont_starve/data/../mods/workshop-584259222/perishable/rog.lua:2
      inst = 113486 - robin_winter(LIMBO) (valid:true)
      frozenfiremult = false
      perishfn = function - @E:/games/stream/steamapps/common/dont_starve/data/scripts/standardcomponents.lua:438
   context = table: 4B24FCC8
   description = nil
   alt_description = nil
E:/games/stream/steamapps/common/dont_starve/data/../mods/workshop-2081254154/modmain.lua:821 in (upvalue) GetEntityInformation (Lua) <778-913>
   entity = 113486 - robin_winter(LIMBO) (valid:true)
   player = 100021 - wilson (valid:true)
   params = table: 4B24FBD8
   assembled = table: 4B24FB10
   player_context = table: 4B24FCC8
   chunks = table: 4B24FC28
   prefab_descriptor = false
   name = perishable
   component = table: 39686E90
   descriptor = table: 4B24CF78
E:/games/stream/steamapps/common/dont_starve/data/../mods/workshop-2081254154/modmain.lua:952 in (upvalue) RequestEntityInformation (Lua) <916-1006>
   entity = 113486 - robin_winter(LIMBO) (valid:true)
   player = 100021 - wilson (valid:true)
   params = table: 4B24FBD8
   insight = table: 13E7F008
E:/games/stream/steamapps/common/dont_starve/data/../mods/workshop-2081254154/scripts/uioverrides.lua:713 in (method) SetString (Lua) <682-830>
   self =
      enabled = true
      focus_target = false
      InsightSetSize = function - @E:/games/stream/steamapps/common/dont_starve/data/../mods/workshop-2081254154/scripts/widgets/text_lib.lua:23
      string = 拾起 便便桶
      shown = false
      SetString = function - @E:/games/stream/steamapps/common/dont_starve/data/../mods/workshop-2081254154/scripts/uioverrides.lua:682
      callbacks = table: 4B5AE6C0
      parent = HoverText
      InsightGetSize = function - @E:/games/stream/steamapps/common/dont_starve/data/../mods/workshop-2081254154/scripts/widgets/text_lib.lua:35
      focus_flow_args = table: 4B5AE5A8
      inst = 109247 -  (valid:true)
      focus = false
      focus_flow = table: 4B5AE6E8
      name = Text
      children = table: 4B5AE698
      Hide = function - @E:/games/stream/steamapps/common/dont_starve/data/../mods/workshop-2081254154/scripts/uioverrides.lua:589
   text = 雪雀
þ: 谋杀
   item = 113486 - robin_winter(LIMBO) (valid:true)
E:/games/stream/steamapps/common/dont_starve/data/scripts/widgets/hoverer.lua:107 in (upvalue) oldOnUpdate (Lua) <23-129>
   self =
      callbacks = table: 4B5AE288
      followhandler = table: 4B5AEA58
      inst = 109246 -  (valid:true)
      focus = false
      children = table: 4B5ADE78
      focus_flow_args = table: 4B5AE418
      focus_target = false
      secondarytext = Text - 
      text = Text - 拾起 便便桶
      insightText = RichText - <null>
      enabled = true
      parent = Controls
      OnUpdate = function - @E:/games/stream/steamapps/common/dont_starve/data/../mods/workshop-2081254154/scripts/uioverrides.lua:674
      name = HoverText
      focus_flow = table: 4B5AE648
      shown = true
      owner = 100021 - wilson (valid:true)
      isFE = false
   using_mouse = true
   str = 雪雀
þ: 谋杀
   colour = table: 4B24F9F8
   secondarystr = nil
E:/games/stream/steamapps/common/dont_starve/data/../mods/workshop-2081254154/scripts/uioverrides.lua:679 in (method) OnUpdate (Lua) <674-680>
   self =
      callbacks = table: 4B5AE288
      followhandler = table: 4B5AEA58
      inst = 109246 -  (valid:true)
      focus = false
      children = table: 4B5ADE78
      focus_flow_args = table: 4B5AE418
      focus_target = false
      secondarytext = Text - 
      text = Text - 拾起 便便桶
      insightText = RichText - <null>
      enabled = true
      parent = Controls
      OnUpdate = function - @E:/games/stream/steamapps/common/dont_starve/data/../mods/workshop-2081254154/scripts/uioverrides.lua:674
      name = HoverText
      focus_flow = table: 4B5AE648
      shown = true
      owner = 100021 - wilson (valid:true)
      isFE = false
   arg = nil
E:/games/stream/steamapps/common/dont_starve/data/scripts/frontend.lua:573 in (method) Update (Lua) <425-593>
   self =
      topblackoverlay = Image - images/global.xml:square.tex
      fade_title_out = false
      ignoreups = table: 15648470
      overlayroot = overlayroot
      save_indicator_time_left = 0
      fade_title_time = 0
      page_repeat_time = 0.25
      OnMouseMove = function - @E:/games/stream/steamapps/common/dont_starve/data/../mods/workshop-2311942869/scripts/components/actionqueuer.lua:350
      helptextbg = Image - images/global.xml:square.tex
      helptext = HelpText
      subtitle = Text - 
      updating_widgets_alt = table: 17D02B70
      title = Text - 
      scroll_repeat_time = -1
      topFadeHidden = false
      gameinterface = 100012 -  (valid:true)
      updating_widgets = table: 1564A1F8
      lastx = 1305
      total_fade_time = 1
      lasty = 6
      autosave_enabled = true
      OnMouseButton = function - @E:/games/stream/steamapps/common/dont_starve/data/../mods/workshop-2311942869/scripts/components/actionqueuer.lua:339
      delayovercb = false
      blackoverlay = Image - images/global.xml:square.tex
      screenstack = table: 156481F0
      num_pending_saves = 0
      fade_delay_time = false
      fade_time = 1.0000012649844
      saving_indicator = UIAnim
      save_indicator_fade_time = 0
      alpha = 0
      tracking_mouse = true
      focus_locked = false
      fade_title_in = false
      screenroot = screenroot
      consoletext = Text - CONSOLE TEXT
      repeat_time = 0.099542799964547
      displayingerror = false
      helptexttext = Text - 
   dt = 0.017001999542117
   k = HoverText
   v = true
E:/games/stream/steamapps/common/dont_starve/data/scripts/update.lua:46 in () ? (Lua) <3-50>
   dt = 0.017001999542117

[00:10:44]: Last called Lua: luaWallUpdate
[00:10:44]: scripts/mainfunctions.lua(960,1) LUA ERROR stack traceback:
        E:/games/stream/steamapps/common/dont_starve/data/scripts/mainfunctions.lua(960,1)	
[00:10:44]: scripts/mainfunctions.lua(961,1) ...rkshop-2081254154/scripts/descriptors/perishable.lua:212: attempt to call method 'GetOwner' (a nil value)
LUA ERROR stack traceback:
    E:/games/stream/steamapps/common/dont_starve/data/../mods/workshop-2081254154/scripts/descriptors/perishable.lua:212 in (upvalue) GetPerishData (Lua) <196-309>
    E:/games/stream/steamapps/common/dont_starve/data/../mods/workshop-2081254154/scripts/descriptors/perishable.lua:328 in (field) Describe (Lua) <311-345>
    E:/games/stream/steamapps/common/dont_starve/data/../mods/workshop-2081254154/modmain.lua:821 in (upvalue) GetEntityInformation (Lua) <778-913>
    E:/games/stream/steamapps/common/dont_starve/data/../mods/workshop-2081254154/modmain.lua:952 in (upvalue) RequestEntityInformation (Lua) <916-1006>
    E:/games/stream/steamapps/common/dont_starve/data/../mods/workshop-2081254154/scripts/uioverrides.lua:713 in (method) SetString (Lua) <682-830>
    E:/games/stream/steamapps/common/dont_starve/data/scripts/widgets/hoverer.lua:107 in (upvalue) oldOnUpdate (Lua) <23-129>
    E:/games/stream/steamapps/common/dont_starve/data/../mods/workshop-2081254154/scripts/uioverrides.lua:679 in (method) OnUpdate (Lua) <674-680>
    E:/games/stream/steamapps/common/dont_starve/data/scripts/frontend.lua:573 in (method) Update (Lua) <425-593>
    E:/games/stream/steamapps/common/dont_starve/data/scripts/update.lua:46 in () ? (Lua) <3-50>
	
[00:10:44]: scripts/frontend.lua(773,1) SCRIPT ERROR! Showing error screen	
```

**Comment by yue-git** [comment]
Language: Chinese
The system information is as follows
```
[00:00:00]: HardwareStats:
  OS                        
    name                      Microsoft Windows 11 专业版
    version                   10.0.22000
    architecture              64 位
```

**Comment by yue-git** [comment]
If information is not enough to help you troubleshoot, please let me know what else you need.

**Comment by penguin0616** [comment]
Thank you for the detailed report and information.
This should be fixed in acb7e967755643638fbd0ba32d0623e6b11d9d7f now.
